articles/container-service/container-service-connect.md

```diff
@@ -71,53 +71,64 @@ For further instructions you can see the [Kubernetes quick start](http://kuberne
 > 
 > 
 
 ## Create an SSH tunnel on Linux or OS X
-The first thing that you do when you create an SSH tunnel on Linux or OS X is to locate the public DNS name of load-balanced masters. To do this, expand the resource group so that each resource is being displayed. Locate and select the public IP address of the master. This will open up a blade that contains information about the public IP address, which includes the DNS name. Save this name for later use. <br />
 
-![Public DNS name](media/pubdns.png)
+1. Get the public DNS name of the load balancer for the master group.
 
-Now open a shell and run the following command where:
+  * To do this in the portal, expand the resource group so that each resource is being displayed. Locate and select the public IP address of the master. This will open up a blade that contains information about the public IP address, which includes the DNS name. Save this name for later use. <br />
 
-**PORT** is the port of the endpoint that you want to expose. For Swarm, this is 2375. For DC/OS, use port 80.  
-**USERNAME** is the user name that was provided when you deployed the cluster.  
-**DNSPREFIX** is the DNS prefix that you provided when you deployed the cluster.  
-**REGION** is the region in which your resource group is located.  
-**PATH_TO_PRIVATE_KEY** [OPTIONAL] is the path to the private key that corresponds to the public key you provided when you created the Container Service cluster. Use this option with the -i flag.
+  ![Public DNS name](media/pubdns.png)
+
+  * Alternatively, use the `az acs show ...` command and look for the "Master Profile:fqdn" property.
+
+2. Run the following command to create an SSH tunnel with one of the cluster
+   masters.  
 
 ```bash
-ssh -L PORT:localhost:PORT -f -N [USERNAME]@[DNSPREFIX]mgmt.[REGION].cloudapp.azure.com -p 2200
+ssh -fNL LOCAL_PORT:localhost:REMOTE_PORT -p 2200 [USERNAME]@[DNSPREFIX]mgmt.[REGION].cloudapp.azure.com
 ```
-> The SSH connection port is 2200--not the standard port 22.
-> 
-> 
+
+**LOCAL_PORT** is the local TCP port to be forwarded over the tunnel.  
+**REMOTE_PORT** is the TCP port on the service side of the tunnel to connect to. For Swarm, set this to 2375. For DC/OS, set this to 80.  
+**USERNAME** is the user name that was provided when you deployed the cluster, by default "azureuser".  
+**DNSPREFIX** is the DNS prefix that you provided when you deployed the cluster.  
+**REGION** is the region in which your resource group is located.  
+
+> Note: The SSH connection port is 2200 and not the standard port 22.
+>
+>
 
 ## DC/OS tunnel
-To open a tunnel to the DC/OS-related endpoints, execute a command that is similar to the following:
+
+To open a tunnel for DC/OS endpoints, execute a command like the following:
 
 ```bash
-sudo ssh -L 80:localhost:80 -f -N azureuser@acsexamplemgmt.japaneast.cloudapp.azure.com -p 2200
+ssh -fNL 8888:localhost:80 azureuser@acsexamplemgmt.japaneast.cloudapp.azure.com -p 2200
```

Comment:
See note above re DC/OS and port 80. We should ***not*** be encouraging people to use a port other than 80. We can add a note suggesting that if they have a problem with port 80 then they can use something like 8888 with the caveat that a few links in the UI will break.